src/main/webapp/dev/js/common/questionRubric.js

```diff
@@ -355,44 +355,33 @@ function highlightRubricCol(index, questionNum, highlight) {
 }
 
 /**
- * Moves the "weights" checkbox to the weight row if it is checked, otherwise
- * moves it to the choice row
+ * Shows the weight row if the "Choices are weighted" checkbox is checked, otherwise
+ * hides the weight row.
  *
- * @param checkbox the "weights" checkbox
+ * @param checkbox the "Choices are weighted" checkbox
  */
-function moveAssignWeightsCheckbox(checkbox) {
-    const $choicesRow = checkbox.closest('thead').find('tr').eq(0);
-    const $weightsRow = checkbox.closest('thead').find('tr').eq(1);
-    const $choicesRowFirstCell = $choicesRow.find('th').first();
+function toggleAssignWeightsRow(checkbox) {
```

Comment:
It's not your fault. The parameter should be named as `$checkbox` to indicate it is a JQuery object.